sources/packages/backend/libs/services/src/confirmation-of-enrollment/confirmation-of-enrollment.service.ts

```diff
@@ -652,7 +655,7 @@ export class ConfirmationOfEnrollmentService {
 
     if (tuitionRemittanceAmount > maxTuitionAllowed) {
       throw new CustomNamedError(
-        "Tuition amount provided should be lesser than both (Actual tuition + Program related costs) and (Canada grants + Canada Loan + BC Loan).",
+        "Tuition amount provided should be lesser than both (Actual tuition + Program related costs + Mandatory fees) and (Canada grants + Canada Loan + BC Loan).",
```

Comment:
👍

Reply:
That'd be good to look for other places to add " + Mandatory fees" like in this file: line 639, `confirmation-of-enrollment.institutions.controller`:  line 251.